src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -40,7 +40,9 @@ public data class YamlConfiguration constructor(
     internal val polymorphismPropertyName: String = "type",
     internal val encodingIndentationSize: Int = 2,
     internal val breakScalarsAt: Int = 80,
-    internal val sequenceStyle: SequenceStyle = SequenceStyle.Block
+    internal val sequenceStyle: SequenceStyle = SequenceStyle.Block,
+    internal val quotedScalarStyle: QuotedScalarStyle = QuotedScalarStyle.DoubleQuoted,
```

Comment:
Should we call this `singleLineStringStyle` instead? That might make the distinction between this and `multiLineStringStyle` clearer.

Reply:
this is what I had originally but it's used for characters and enums also, your call

Reply:
Let's go with `singleLineStringStyle`.